client/my-sites/plan-features/header.jsx

```diff
@@ -95,13 +96,27 @@ class PlanFeaturesHeader extends Component {
 		);
 	}
 
+	getDiscountTooltipMessage() {
+		const { currencyCode, currentSitePlan, translate, rawPrice } = this.props;
+
+		if ( currentSitePlan.productSlug === PLAN_FREE ) {
+			return translate( 'Price for the next 12 months' );
+		}
+
+		const price = formatCurrency( rawPrice, currencyCode );
+
+		return translate(
+			'We’ll deduct the cost of your current plan from the full price (%(price)s)',
```

Comment:
I wonder if this is still clear. Based on the comments yesterday, I got the sense that people thought that this would be their new annual price. What if it's reworded to something like this:

`We’ll deduct the cost of your current plan from the full price (%(price)s) for the rest of this year.`

cc @anneforbush

Reply:
Good point. Slight modification to that because "this year" can be a bit ambiguous:

"We'll deduct the cost of your current plan from the full price (%(price)s) for the next 12 months."